src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/config/Config.java

```diff
@@ -305,36 +281,25 @@ public void setValues(JSONObject formData) {
             numberOfSendingWorkerThreads = DEFAULT_NR_OF_SENDING_WORKER_THREADS;
         }
 
-        gerritBuildStartedVerifiedValue = formData.optInt(
-                "gerritBuildStartedVerifiedValue",
-                DEFAULT_GERRIT_BUILD_STARTED_VERIFIED_VALUE);
-        gerritBuildStartedCodeReviewValue = formData.optInt(
-                "gerritBuildStartedCodeReviewValue",
-                DEFAULT_GERRIT_BUILD_STARTED_CODE_REVIEW_VALUE);
-        gerritBuildSuccessfulVerifiedValue = formData.optInt(
-                "gerritBuildSuccessfulVerifiedValue",
-                DEFAULT_GERRIT_BUILD_SUCCESSFUL_VERIFIED_VALUE);
-        gerritBuildSuccessfulCodeReviewValue = formData.optInt(
-                "gerritBuildSuccessfulCodeReviewValue",
-                DEFAULT_GERRIT_BUILD_SUCCESSFUL_CODE_REVIEW_VALUE);
-        gerritBuildFailedVerifiedValue = formData.optInt(
-                "gerritBuildFailedVerifiedValue",
-                DEFAULT_GERRIT_BUILD_FAILURE_VERIFIED_VALUE);
-        gerritBuildFailedCodeReviewValue = formData.optInt(
-                "gerritBuildFailedCodeReviewValue",
-                DEFAULT_GERRIT_BUILD_FAILURE_CODE_REVIEW_VALUE);
-        gerritBuildUnstableVerifiedValue = formData.optInt(
-                "gerritBuildUnstableVerifiedValue",
-                DEFAULT_GERRIT_BUILD_UNSTABLE_VERIFIED_VALUE);
-        gerritBuildUnstableCodeReviewValue = formData.optInt(
-                "gerritBuildUnstableCodeReviewValue",
-                DEFAULT_GERRIT_BUILD_UNSTABLE_CODE_REVIEW_VALUE);
-        gerritBuildNotBuiltVerifiedValue = formData.optInt(
-                "gerritBuildNotBuiltVerifiedValue",
-                DEFAULT_GERRIT_BUILD_NOT_BUILT_VERIFIED_VALUE);
-        gerritBuildNotBuiltCodeReviewValue = formData.optInt(
-                "gerritBuildNotBuiltCodeReviewValue",
-                DEFAULT_GERRIT_BUILD_NOT_BUILT_CODE_REVIEW_VALUE);
+        if (formData.isEmpty()) {
```

Comment:
I think we are saying the same thing...

We make use of the `forData.isEmpty` construct for BOTH Verified and CodeReview cases.